### PR TITLE
Added a phpcs.xml.dist file

### PR DESCRIPTION
**Pull request by gmponos**
## Description

I have added a phpcs.xml.dist file for codesniffer.

This dist file works in the same logic as phpunit.xml.dist. Developers can add their own custom one.

I don't know exactly the version that codesniffer package added this feature. Going through the releases I think it is on 2.5.0. So it might be better if we wait for this #129 to be merged first.

## Motivation and context

Adding an XML with rules is more friendly rather than running through the command line the rules that you need.

## How has this been tested?

I have run composer update and executed the commands of composer.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [x] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 build
 composer.lock
 vendor
+phpcs.xml
+phpunit.xml
```
Comment (gmponos) [inline .gitignore]: Is it wrong that I have also added phpunit.xml here?

Comment (frankdejonge) [reply .gitignore]: @gmponos nope, that's fine 👍

**Comment by gmponos** [comment]
I forgot to add the link with the 2.5.0 release changelog of codesniffer

here it is https://github.com/squizlabs/PHP_CodeSniffer/releases/tag/2.5.0

**Comment by frankdejonge** [comment]
Cheers @gmponos 👍

**Pull request merged**